### PR TITLE
docs: update docker compose setup for api

### PR DESCRIPTION
**Pull request by ayushnvs**
### Pre-requisites

- [x] I have gone through the Contributing guidelines for [Submitting a Pull Request (PR)](../../CONTRIBUTING.md#submitting-a-pull-request-pr) and ensured that this is not a duplicate PR.
- [ ] I have performed preliminary testing using the [test suite](../../apps/api/OsmoX.postman_collection.json) to ensure that any existing features are not impacted and any new features are working as expected.
- [x] I have updated the required [api docs](../../apps/api/docs/) as applicable.
- [ ] I have added/updated test cases to the [test suite](../../apps/api/OsmoX.postman_collection.json) as applicable

### PR Details

PR details have been updated as per the given format (see below)

- [x] PR title adheres to the format specified in guidelines (e.g., `feat: add admin login endpoint`)
- [x] Description has been added
- [ ] Related changes have been added (optional)
- [ ] Screenshots have been added (optional)
- [ ] Query request and response examples have been added (as applicable, in case added or updated)
- [x] Documentation changes have been listed (as applicable)
- [ ] Test suite output is added (as applicable)
- [ ] Pending actions have been added (optional)
- [ ] Any other additional notes have been added (optional)

### Additional Information

- [x] Appropriate label(s) have been added (`ready for review` should be added if the PR is ready to be reviewed)
- [ ] Assignee(s) and reviewer(s) have been added (optional)

---

**Description:**

Updated .env.example file to add environment variable *COMPOSE_PROJECT_NAME*
Updated the related documents *production-setup.md* and *development-stup.md*

**Related changes:**

N/A

**Screenshots:**

N/A

**Query request and response:**

N/A

**Documentation changes:**

- In *production-setup.md*, added environment variable *COMPOSE_PROJECT_NAME* in section Server Configuration step 1.
- In *development-setup.md*, added environment variable *COMPOSE_PROJECT_NAME*  in section Getting Started step 3.

**Test suite output:**

N/A

**Pending actions:**

N/A

**Additional notes:**

N/A

## Summary by CodeRabbit

- **New Features**
	- Added a new temporary environment variable `COMPOSE_PROJECT_NAME` to specify the project name across different setups.

- **Documentation**
	- Updated development and production setup documentation to include the new `COMPOSE_PROJECT_NAME` environment variable.

### CODE REVIEW

File: apps/api/.env.example [hunk 1]
```diff
@@ -16,4 +16,7 @@ MARIADB_DOCKER_PORT=3307
 # Redis configuration
 REDIS_HOST=
 REDIS_PORT=
-REDIS_DOCKER_PORT=6397
\ No newline at end of file
+REDIS_DOCKER_PORT=6397
+
+# TEMP
+COMPOSE_PROJECT_NAME=project-name  # Add your project name here.
```
[the remark this answers is not in the document]
Comment (ayushnvs) [reply apps/api/.env.example]: Updated `project-name` to `osmo-x`
Updated `# TEMP` to `# Docker env`

File: apps/api/docs/development-setup.md [hunk 2]
```diff
@@ -60,6 +60,9 @@ sudo systemctl status mariadb
    REDIS_HOST=127.0.0.1 # use value as osmox-redis in docker
    REDIS_PORT=6379
    REDIS_DOCKER_PORT=6397 # (required only if using docker)
+
+   # TEMP
+   COMPOSE_PROJECT_NAME=osmo-x  # Add your project name here.
```
[the remark this answers is not in the document]
Comment (ayushnvs) [reply apps/api/docs/development-setup.md:65]: Updated to default value "osmo-x".

File: apps/api/docs/production-setup.md [hunk 3]
```diff
@@ -38,6 +38,9 @@ Make sure Redis and MariaDB server are up and running.
   REDIS_HOST=127.0.0.1 # use value as osmox-redis in docker
   REDIS_PORT=6379
   REDIS_DOCKER_PORT=6397 # (required only if using docker)
+
+  # TEMP
+  COMPOSE_PROJECT_NAME=osmo-x  # Add your project name here.
```
[the remark this answers is not in the document]
Comment (ayushnvs) [reply apps/api/docs/production-setup.md:43]: Updated to default value "osmo-x".

File: apps/api/docs/development-setup.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (LakshayaT) [reply apps/api/docs/development-setup.md:65]: This is good enough for the time being as we want it to be the default value only, and we have added a comment already to set the project name.

**Review by LakshayaT — APPROVED** [review]

**Pull request merged**